code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -2211,12 +2218,12 @@ func (srh *StorageRestHandler) getTransactionByFilter(w http.ResponseWriter, r *
 //	400:
 //	500:
 func (srh *StorageRestHandler) getTransactionHashesByFilter(w http.ResponseWriter, r *http.Request) {
-	var (
-		lookUpHash  = r.URL.Query().Get("look-up-hash")
-		writeMarker = r.URL.Query().Get("name")
-		contentHash = r.URL.Query().Get("content-hash")
-	)
-
+	fileIDStr := r.URL.Query().Get("fileid")
+	fileID, err := strconv.ParseInt(fileIDStr, 10, 64)
+	if err != nil || fileID <= 0 {
+		common.Respond(w, r, nil, common.NewErrBadRequest("Invalid file id."))
```

Comment:
Let's use consistent error message, and avoid capital error message, so pleases use 'invalid file id'. And perhaps have a function to parse and get the `file_id` param to avoid duplicate work.